lfs/credentials.go

```diff
@@ -51,7 +51,8 @@ func getCredsForAPI(req *http.Request) (Creds, error) {
 }
 
 func getCredURLForAPI(req *http.Request) (*url.URL, error) {
-	apiUrl, err := Config.ObjectUrl("")
```

Comment:
I dropped ObjectUrl because it seems defunct except for this check, where it doesn't even use an oid. Seems it's just looking for a base api url to look up the creds for and Endpoint does that.